Foundation/NSSwiftRuntime.swift

```diff
@@ -81,12 +81,12 @@ internal func _CFZeroUnsafeIvars<T>(_ arg: inout T) {
 }
 
 @_cdecl("__CFSwiftGetBaseClass")
-internal func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
+public func __CFSwiftGetBaseClass() -> UnsafeRawPointer {
     return unsafeBitCast(__NSCFType.self, to:UnsafeRawPointer.self)
 }
 
 @_cdecl("__CFInitializeSwift")
-internal func __CFInitializeSwift() {
+public func __CFInitializeSwift() {
```

Comment:
However we set in the project scope variable (with no override) `CLANG_LINK_OBJC_RUNTIME` to `NO`